applications/GeoMechanicsApplication/custom_conditions/README.md

```diff
@@ -0,0 +1,141 @@
+# Microclimate Boundary Condition
+A microclimate boundary condition refers to the specific set of environmental factors and conditions that influence the climate within a small, localized area, known as a microclimate. Unlike the broader regional or global climate, microclimates are unique climatic conditions that can vary significantly within a relatively small space.
+
+Microclimate boundary conditions are determined by factors such as:
+
+- Air temperature
+- Solar radiation
+- Humidity
+- Precipitation
+- Wind speed
+
+## Radiation
+The net radiation $R_n$ $\mathrm{[W/m^2]}$ is calculated by:
+
+$$ R_n = R^s + R^{l,ab} - R^{l,em} $$
+
+where
+
+- $R^s$ = in-coming short wave radiation $\mathrm{[W/m^2]}$
+- $R^{l,ab}$ = absorbed long wave radiation $\mathrm{[W/m^2]}$
+- $R^{l,em}$ = emitted long wave radiation $\mathrm{[W/m^2]}$
+
+The incoming short wave radiation is defined by
+
+$$ R^s = \left(1 - \alpha \right) R_{g} $$
+
+where
+
+- $\alpha$ = Albedo cover or vegetation coefficient  $\mathrm{[-]}$
+- $R_{g}$ = Short wave radiation  $\mathrm{[W/m^2]}$ 
+
+$R_g$ is a user difined parameter which is usually provided as a "Time-Radiation" table.
+
+The absorbed long wave radiation is:
+
+$$ R^{l,ab}=  \epsilon \sigma \left( T_{at} +273.15 \right)^4 $$
+
+where
+
+- $\epsilon$ = effective emissivity  $\mathrm{[-]}$
+- $\sigma$ = Stefan Boltzmann's constant  $\mathrm{[5.67 \cdot 10^{-8}  W/m^{2 \circ}C^4]}$
+- $T_{at}$ = atmospheric temperature $\mathrm{[^{\circ}C]}$
+
+And the emitted long wave radiation,
+
+$$ R^{l,em} = \sigma \left( T_{ss} +273.15 \right)^4 $$
+
+where $T_{ss}$ $\mathrm{[^{\circ}C]}$ is land surface temperature.
+
+## Surface Heat Storage
+The surface heat storage term $Q_s$ $\mathrm{[W/m^2]}$ is calculated by,
+
+$$ Q_s =  a_1 R_n(t+\Delta t) + a_2 \frac{R_n(t+\Delta t) - R_n(t)}{\Delta t} + a_3 $$
+
+where $a_1$ $\mathrm{[-]}$, $a_2$ $\mathrm{[s]}$ and $a_3$ $\mathrm{[W/m^2]}$ are user defined variables.
```

Comment:
Is there any expected range for $a_n$ (e.g. between 0 and 1)? If so it might be a good idea to mention it here and add it to the code (in a separate PR) if it's not included there.